src/meta/test/ActiveHostsMergeTest.cpp

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Maybe just put the ut in ActiveHostsManTest?

Reply:
yes!